website/app-templates/smarty/navbar.tpl

```diff
@@ -31,7 +31,9 @@
 
             <ul class="nav navbar-nav">
                 <li><a href="{'home'|alias}"><span class="glyphicon glyphicon-home" aria-hidden="true"></span> {t}Home{/t}</a></li>
+                {if $news}
```

Comment:
I did not thought about that before, but in this context, `$news` only exists in some pages (`home` at least). We should revert this, and add proper message in `news_list.tpl` for the case where there is no news to list.

Reply:
okay 😀

Reply:
(I've already prepared a fix for it)